test/unit/spec/narrative_core/kbaseNarrativeStagingDataTab-spec.js

```diff
@@ -138,13 +159,12 @@ define(['jquery', 'kbaseNarrativeStagingDataTab', 'base/js/namespace'], (
             jasmine.clock().install();
             await stagingWidget.render();
             // run a bunch of triggers, should only call render on the staging area once
-            spyOn(stagingWidget.stagingAreaViewer, 'render');
-            for (let i = 0; i < 100; i++) {
+            spyOn(stagingWidget.stagingAreaViewer, 'updateView');
+            for (let i = 0; i < 50; i++) {
                 stagingWidget.uploadWidget.dropzone.emit('complete', { name: 'foo', size: 12345 });
             }
             jasmine.clock().tick(stagingWidget.minRefreshTime + 100);
-            expect(stagingWidget.stagingAreaViewer.render.calls.count()).toEqual(1);
-
+            expect(stagingWidget.stagingAreaViewer.updateView.calls.count()).toEqual(2);
```

Comment:
After some changes, this calls once on the first request as a result of the dropzone event, then once more after the forced clock tick.